crates/worker/src/partition/services/non_deterministic/mod.rs

```diff
@@ -74,20 +67,18 @@ impl ServiceInvoker {
         &mut self,
         full_invocation_id: FullInvocationId,
         method: &str,
-        span_context: ServiceInvocationSpanContext,
+        _span_context: ServiceInvocationSpanContext,
```

Comment:
Could this span_context be removed?

Reply:
This was actually a bad removal, I added a new commit that uses this span_context in the idempotent invoker to link the invoked component